[Camera] Crash when selecting a cloud-stored Google Photos image

Description of the problem:
When selecting an image from the camera's file picker option in Google Photos, if the photo is not saved on the device, the application crashes after selection.

Affected platform
- [ X ] Android
- [ ] iOS
- [ ] electron
- [ ] web

OS of the development machine
- [ ] Windows
- [ X ] macOS
- [ ] linux

Other information:
Affected intent URIs contain "mediakey" instead of "content".
A modified version of [this fix](https://github.com/gbaccetta/plugins/blob/9f1260cc05ead4340ccf4c8f15dabd665b6aaa24/packages/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/FileUtils.java#L131) works on my test Pixel 1.
That fix was modded to convert the path back into a URI.

I'm attaching my hacked test Camera.java for reference.
[Camera.java.zip](https://github.com/ionic-team/capacitor/files/2842989/Camera.java.zip)

Stacktrace:
```
E/AndroidRuntime: FATAL EXCEPTION: main
    Process: APP.IDENTIFIER, PID: 24819
    java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=9003, result=-1, data=Intent { dat=content://com.google.android.apps.photos.contentprovider/0/1/mediakey:/local%3A917a069c-99ea-476a-98df-695d2c3dda21/ORIGINAL/NONE/425538234 flg=0x1 clip={text/uri-list U:content://com.google.android.apps.photos.contentprovider/0/1/mediakey%3A%2Flocal%253A917a069c-99ea-476a-98df-695d2c3dda21/ORIGINAL/NONE/425538234} }} to activity {APP.IDENTIFIER/APP.IDENTIFIER.MainActivity}: java.lang.IllegalArgumentException: filename cannot be null
        at android.app.ActivityThread.deliverResults(ActivityThread.java:4496)
        at android.app.ActivityThread.handleSendResult(ActivityThread.java:4538)
        at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:49)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1906)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6863)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: java.lang.IllegalArgumentException: filename cannot be null
        at android.media.ExifInterface.<init>(ExifInterface.java:1330)
        at com.getcapacitor.plugin.camera.ImageUtils.getExifData(ImageUtils.java:124)
        at com.getcapacitor.plugin.Camera.returnResult(Camera.java:323)
        at com.getcapacitor.plugin.Camera.processPickedImage(Camera.java:267)
        at com.getcapacitor.plugin.Camera.handleOnActivityResult(Camera.java:450)
        at com.getcapacitor.Bridge.onActivityResult(Bridge.java:697)
        at com.getcapacitor.BridgeActivity.onActivityResult(BridgeActivity.java:215)
        at android.app.Activity.dispatchActivityResult(Activity.java:7476)
        at android.app.ActivityThread.deliverResults(ActivityThread.java:4489)
        at android.app.ActivityThread.handleSendResult(ActivityThread.java:4538) 
        at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:49) 
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108) 
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1906) 
        at android.os.Handler.dispatchMessage(Handler.java:106) 
        at android.os.Looper.loop(Looper.java:193) 
        at android.app.ActivityThread.main(ActivityThread.java:6863) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858) 
2019-02-07 15:57:21.071 30131-30131/APP.IDENTIFIER E/AndroidRuntime: FATAL EXCEPTION: main
    Process: APP.IDENTIFIER, PID: 30131
    java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=9003, result=-1, data=Intent { dat=content://com.google.android.apps.photos.contentprovider/0/1/mediakey:/local%3A917a069c-99ea-476a-98df-695d2c3dda21/ORIGINAL/NONE/280467212 flg=0x1 clip={text/uri-list U:content://com.google.android.apps.photos.contentprovider/0/1/mediakey%3A%2Flocal%253A917a069c-99ea-476a-98df-695d2c3dda21/ORIGINAL/NONE/280467212} }} to activity {APP.IDENTIFIER/APP.IDENTIFIER.MainActivity}: java.lang.IllegalArgumentException: filename cannot be null
        at android.app.ActivityThread.deliverResults(ActivityThread.java:4496)
        at android.app.ActivityThread.handleSendResult(ActivityThread.java:4538)
        at android.app.servertransaction.ActivityResultItem.execute(ActivityResultItem.java:49)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1906)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6863)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: java.lang.IllegalArgumentException: filename cannot be null
        at android.media.ExifInterface.<init>(ExifInterface.java:1330)
        at com.getcapacitor.plugin.camera.ImageUtils.getExifData(ImageUtils.java:124)
        at com.getcapacitor.plugin.Camera.returnResult(Camera.java:325)
        at com.getcapacitor.plugin.Camera.processPickedImage(Camera.java:269)
        at com.getcapacitor.plugin.Camera.handleOnActivityResult(Camera.java:452)
        at com.getcapacitor.Bridge.onActivityResult(Bridge.java:697)
        at com.getcapacitor.BridgeActivity.onActivityResult(BridgeActivity.java:215)
        at android.app.Activity.dispatchActivityResult(Activity.java:7476)
        at android.app.ActivityThread.deliverResults(ActivityThread.java:4489)
```
Capacitor version: 1.0.0-beta.17

Steps to reproduce:
- Invoke the camera plugin's getPhoto method.
- Select `From Photos`
- Choose Google Photos (if applicable)
- Select an image from the gallery that is not stored on the device.

--- a/android/capacitor/src/main/java/com/getcapacitor/FileUtils.java
+++ b/android/capacitor/src/main/java/com/getcapacitor/FileUtils.java
@@ -131,7 +131,6 @@ public class FileUtils {
       // Return the remote address
       if (isGooglePhotosUri(uri))
         return uri.getLastPathSegment();
-
       return getDataColumn(context, uri, null, null);
     }
     // File
@@ -154,7 +153,7 @@ public class FileUtils {
    */
   private static String getDataColumn(Context context, Uri uri, String selection,
                                       String[] selectionArgs) {
-
+    String path = null;
     Cursor cursor = null;
     final String column = "_data";
     final String[] projection = {
@@ -166,7 +165,7 @@ public class FileUtils {
           null);
       if (cursor != null && cursor.moveToFirst()) {
         final int index = cursor.getColumnIndexOrThrow(column);
-        return cursor.getString(index);
+        path = cursor.getString(index);
       }
     } catch (IllegalArgumentException ex) {
       return getCopyFilePath(uri, context);
@@ -174,7 +173,10 @@ public class FileUtils {
       if (cursor != null)
         cursor.close();
     }
-    return null;
+    if (path == null) {
+      return getCopyFilePath(uri, context);
+    }
+    return path;
   }
 
   private static String getCopyFilePath(Uri uri, Context context) {

